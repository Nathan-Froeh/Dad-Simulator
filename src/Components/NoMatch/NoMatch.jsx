import React from 'react';
import {NavLink} from 'react-router-dom';


const NoMatch = () => {
  return(
    <div>
      <nav>
        <NavLink to={'/menu'}>
          <p>Menu</p>
        </NavLink>
        <h1>Virtual Dad Simulator</h1>
      </nav>
      <h2>Page does not exist</h2>
    </div>
  )
}

export default NoMatch;