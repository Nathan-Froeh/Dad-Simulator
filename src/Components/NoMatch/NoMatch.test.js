import React from 'react';
import {shallow} from 'enzyme';
import NoMatch from './NoMatch';

describe('NoMatch', () => {
  const wrapper = shallow(<NoMatch/>)

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})