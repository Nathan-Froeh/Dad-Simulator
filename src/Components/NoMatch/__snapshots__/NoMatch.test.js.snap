// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should match snapshot 1`] = `ShallowWrapper {}`;
